QUARKUS-6258.md

```diff
@@ -64,10 +75,14 @@ Important notes:
 Upstream uses gpt-4o-mini for testing, which costs roughly 1.5 times more, and the latest version of it (gpt-4.1-mini) costs four times more. It seems, that we may want to request ~10 USD worth of credits for the testing, and adjust it in the future. Another benefit for using cloud infrastructure is that it would be easier to use it during a move of infrastructure from Red Hat to IBM. 
 Watson.x foundational models cost the same[3], but there is a free tier (with up to 50,000 tokens per month) which we may fit in and given, that we are part of IBM now, we may be able to get access from them.
 
+## Non-automated testing
+
+See https://github.com/quarkus-qe/quarkus-test-plans/blob/main/QUARKUS-6257.md#additional-requirements-for-full-support for details.
```

Comment:
Why is this in non-automated testing? We need to test all those requirements automatically (except #7, which is an attestation) if this is to be fully supported (and we do for all the supported extensions already).

Reply:
I doubt we can check upstream guides, existence of CVE detection by tools and confidence of dev team automatically.

Reply:
I suppose that's right, but it's still really unclear considering most of those points are not automated

Reply:
That is why I put these in non-automated testing section. Since we agree on that, am I correct to assume this matter to be resolved?